english/explore-outside/talk-child/talk-more/nice-talk/walk-away/walk-away.md

```diff
@@ -0,0 +1,2 @@
+Unsure how to react, you decide to simply walk away from this dilemma.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.